modules/customization/proc-customize-rhdh-sidebar-menuitems.adoc

```diff
@@ -0,0 +1,54 @@
+[id='proc-customize-rhdh-sidebar-menuitems_{context}']
+= Customizing the sidebar navigation menu items for your {product-short} instance
+
+You can customize the sidebar menu items in your `app-config-rhdh.yaml` file using the following attributes:
+
+[source,yaml]
+----
+dynamicPlugins:
+  frontend:
+    <package_name>: # same as `scalprum.name` key in plugin's `package.json`
+      menuItems: # optional, allows you to configure plugin menu items in the main sidebar navigation
+        <menu_item_name>: # unique name in the plugin menu items list <1>
+          icon: fooIcon # optional, same as `menuItem.icon` in `dynamicRoutes` <2>
+          title: Foo Plugin Page # optional, same as `menuItem.text` in `dynamicRoutes` <3>
+          priority: 10 # optional, defines the order of menu items in the sidebar <4>
+          parent: favorites # optional, defines parent-child relationships for nested menu items <5>
+----
+
+<1> This attribute represents a unique name in the main sidebar navigation. It can denote either a standalone menu item or a parent menu item. If the attribute represents a plugin menu item, If this attribute represents a plugin menu item, the name of the attribute must match with the corresponding path in `dynamicRoutes.path`. For example, if `dynamicRoutes.path` defines `path: /my-plugin`, then `menu_item_name` must be defined as `my-plugin`.
+
+<2> This optional attribute specifies the icon for the menu item. You can use default icons or extend the icon set with dynamic plugins. {product-short} also provides additional icons in its internal library. If the icon is already defined in the `dynamicRoutes` configuration under `menuItem.icon`, it can be removed from the in the `menuItems` configuration.
```

Comment:
>{product-short} also provides additional icons in its internal library.

Should we show what internal icons are available to use?

Reply:
Yes, that would be great! However, do we have an alternative link to [CommonIcons.tsx](https://github.com/janus-idp/backstage-showcase/blob/main/packages/app/src/components/DynamicRoot/CommonIcons.tsx)? This link points to the showcase repository, and we typically avoid including upstream links in our downstream documentation.

Reply:
I don't have an alternative link to the actual file, but we can provide some info as below. Feel free to change the format of it.
```
  home: from '@mui/icons-material/HomeOutlined'
  group: from '@mui/icons-material/People'
  category: from '@mui/icons-material/CategoryOutlined'
  extension: from '@mui/icons-material/ExtensionOutlined'
  school: from '@mui/icons-material/SchoolOutlined'
...
```

Reply:
Also add this: If icons are not available in the library, `<svg>` and `html` image icons are also supported. Such as
```
icon: <svg width="20px" height="20px" viewBox="0 0 512 512" xmlns="http://www.w3.org/2000/svg" fill="#ffffff">...</svg>
```
and
```
icon: https://img.icons8.com/ios-glyphs/20/FFFFFF/shop.png
```

Reply:
Done